src/app_service/service/accounts/service.nim

```diff
@@ -46,6 +49,53 @@ proc getImportedAccount*(self: Service): GeneratedAccountDto =
 proc isFirstTimeAccountLogin*(self: Service): bool =
   return self.isFirstTimeAccountLogin
 
+# Remove extra copied keystore that are not needed (can be remove 1st jan 2023)
+proc deleteExtraKeyStoreFile*(self: Service) =
+  let accounts = status_account.getAccounts().result
+  var deleteCandidates: seq[string] = @[]
+  for path in walkFiles(self.keyStoreDir & "*"):
+    var found = false
+    for account in accounts.getElems():
+      let address = account{"address"}.getStr.replace("0x", "")
+      if path.endsWith(address):
+        found = true
+        break
+
+    if not found:
+      deleteCandidates.add(path)
+
+  if len(deleteCandidates) == 2:
+    return
```

Comment:
@alaibe just curious... why do we return here of there are only 2 delete candidates?

Reply:
I will add a comment to make it clear, this is in case you never imported any account in the wallet, there would be 4 files in the keystore, 2 of them will be considered orphan, but we should keep them. This is why the 2